views/includes/list/update-review-bo-mo.html

```diff
@@ -0,0 +1,81 @@
+<div class="govuk-summary-card">
+    <div class="govuk-summary-card__title-wrapper">
+        <h2 class="govuk-summary-card__title">What you have reviewed</h2>
+    </div>
+    <div class="govuk-summary-card__content">
+        <dl class="govuk-summary-list">
+            {% for boi in beneficial_owners_individual %}
+                {% if boi.ch_reference %}
+                    <div class="govuk-summary-list__row">
+                        <dt class="govuk-summary-list__key">
+                            {{ boi.first_name }} {{ boi.last_name }}
+                        </dt>
+                        <dd class="govuk-summary-list__value">
+                            Individual beneficial owner
+                        </dd>
+                        <dd class="govuk-summary-list__value summary-status" aria-label="status">
+                            {% if boi.ceased_date|length === 0 %}
+                                Active
+                            {% else %}
+                                Ceased
+                            {% endif %}
+                        </dd>
+                        <dd class="govuk-summary-list__actions">
+                            <a class="govuk-link" href="{{ reviewBeneficialOwnerIndividualUrl }}{{ indexQueryParam }}{{ beneficial_owners_individual.indexOf(boi) }}">
+                                Change<span class="govuk-visually-hidden"> {{ boi.first_name }} {{ boi.last_name }} - Individual beneficial owner</span>
+                            </a>
+                        </dd>
+                    </div>
+                {% endif %}
+            {% endfor %}
+            {% for boo in beneficial_owners_corporate %}
```

Comment:
Jitin has a filter for this in his review. Please feel free to steal! https://github.com/companieshouse/overseas-entities-web/pull/873

Reply:
Tried this but had to remove in the end, see below